src/main/features/response/form/models/statement-of-means/dependants.ts

```diff
@@ -6,32 +6,31 @@ import { toNumberOrUndefined } from 'common/utils/numericUtils'
 import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
 
 export class ValidationErrors {
-  static readonly UNDER_11_REQUIRED: string = 'Enter a number of children under 11'
-  static readonly BETWEEN_11_AND_15_REQUIRED: string = 'Enter a number of children between 11 and 15'
-  static readonly BETWEEN_16_AND_19_REQUIRED: string = 'Enter a number of children between 19 and 19'
+  static readonly ENTER_AT_LEAST_ONE: string = 'Enter a number for at least one field'
 }
 
 export class Dependants implements Serializable<Dependants> {
 
   @IsDefined({ message: GlobalValidationErrors.YES_NO_REQUIRED })
   hasAnyChildren: boolean
 
-  @ValidateIf(o => o.hasAnyChildren === true)
-  @IsDefined({ message: ValidationErrors.UNDER_11_REQUIRED })
-  @IsInt({ message: GlobalValidationErrors.NUMBER_REQUIRED })
-  @Min(0, { message: GlobalValidationErrors.NUMBER_REQUIRED })
+  @ValidateIf(o => o.hasAnyChildren === true && !o.under11 && !o.between11and15 && !o.between16and19)
+  @IsInt({ message: ValidationErrors.ENTER_AT_LEAST_ONE })
```

Comment:
Do we need this. Usually we make input fields of type `number` and deserialisation method deserialises value to proper type. That solution suggests there expected type is a number but it might not be a number. There is something wrong with the deserialisation.

Reply:
It's not about 'type of', it's about 1, 2, 3.. vs 1.1, 3.14 etc :)

Reply:
I see you trying to prevent numbers with decimal points.

Reply:
It will never be a number as this is a field to reflect cross-field validation errors only. It's not displayed (or even hidden in html code). IMO works good and I have thought about other options (like eg. doing it in routes) and I find it much better - validation is done by model, encapsulated, easier to unit test, tested in the same test file).

Reply:
Solution would be:

1. wrapping numeric values with some internal class
2. implementing custom validator for the internal class ensuring at least one value within instance is populated
3. applying validator to the property of newly created wrapper type